src/model/Position.java

```diff
@@ -1,42 +0,0 @@
-package model;
-
-public class Position {
-	public int x, y;
-
-	public Position() {
-		x = 0;
-		y = 0;
-	}
-
-	public Position(int x, int y) {
-		this.x = x;
-		this.y = y;
-	}
-
-	public Position(Position p) {
-		x = p.x;
-		y = p.y;
-	}
-
-	@Override
-	public boolean equals(Object obj) {
-		if (obj == null)
-			return false;
-		if (!(obj instanceof Position))
-			return false;
-		Position p = (Position) obj;
-		return this.x == p.x && this.y == p.y;
-	}
-
-	@Override
-	public int hashCode() {
-		int result = x;
-		result = 31 * result + y;
```

Comment:
Nice :). Maybe it's there in the 4th volume.
Still, the issue in the next comment remains valid. If the "Position" could somehow be customisable in that "31", then using the columns count instead would do...
('cause there's no single answer to The Question ;) )